fluent/fluent.go

```diff
@@ -338,13 +368,37 @@ func (f *Fluent) Close() (err error) {
 // appendBuffer appends data to buffer with lock.
 func (f *Fluent) appendBuffer(msg *msgToSend) error {
 	select {
-	case f.pending <- msg:
+	case f.pending <- bufferInput{msg: msg}:
 	default:
 		return fmt.Errorf("fluent#appendBuffer: Buffer full, limit %v", f.Config.BufferLimit)
 	}
 	return nil
 }
 
+// appendBufferWithFeedback appends data to buffer and waits for the result
```

Comment:
`appendBufferWithFeedback` did you update the name after writing this comment?

Reply:
I did, will change the comment :)